image/discourse_test/Dockerfile

```diff
@@ -33,7 +33,7 @@ RUN cd /var/www/discourse &&\
     sudo -u discourse yarn cache clean
 
 RUN cd /var/www/discourse && sudo -E -u discourse -H bundle exec rake plugin:install_all_official &&\
-    sudo -E -u discourse -H bundle exec rake plugin:install_all_gems &&\
+    sudo -E -u discourse -H -e LOAD_PLUGINS=1 bundle exec rake plugin:install_all_gems &&\
```

Comment:
eading `man sudo`, `-e` is:

> --edit Edit one or more files instead of running a command. (…)

Reply:
Oops I thought I was dealing with a docker container.